### PR TITLE
added community api calls

### PR DESCRIPTION
**Pull request by vasdhara**
#23 
Added query params for the react client to differentiate between a community api call and mobile home api call
added boolean in results to help differentiate
added type definitions for community results
added api call to different endpoint for communities

### CODE REVIEW

**Comment by vasdhara** [comment]
@jimjeffers restructured files as discussed.

File: src/types.ts [hunk 1]
```diff
@@ -98,17 +107,53 @@ export interface IUnparsedMobileHome {
   longitude: string
   manufacturerName: string
   modelType: string
+  photoSmall: string
   photoLarge: string
   rentalPrice: number
   url: string
 }
 
+export interface ICommunity {
```
Comment (jeremy-loss) [inline src/types.ts]: We have some common fields that I would like to consolidate. To do this I think we should do the following:

interface IMHBOListing {
   id: number,
   address: IAddress,
   isCommunity: boolean,
   url
}

interface ICommunity  extends IMHBOListing {
... remove fields already coming from IMHBOListing
}

interface IMobileHome  extends IMHBOListing {
... remove fields already coming from IMHBOListing
}

For now we can keep photoLarge and photoSmall in the child interfaces. We can always rearrange this later. For performance reasons I would like to prepare for the initial radius search to only return IMHBOListing data and the details be returned and cached based on what needs to be displayed to the user at that time.

File: src/types.ts [hunk 2]
```diff
@@ -98,17 +107,53 @@ export interface IUnparsedMobileHome {
...
+  address: IAddress
+  country: string | null
+  county: string
```
Comment (jeremy-loss) [inline src/types.ts]: Country and county are already part of address so I don't think they are needed.

**Review by jeremy-loss — CHANGES_REQUESTED** [review]

**Review by jeremy-loss — APPROVED** [review]
This looks good. I will try and get it pushed to npm shortly.

**Pull request merged**